test/ecoli_flux.jl

```diff
@@ -18,7 +18,9 @@ end
 
     @test typeof(mdl) == Model
 
-    @test_logs (:error, "SBML reported error: File unreadable.") @test_throws AssertionError readSBML(sbmlfile * ".does.not.really.exist")
+    @test_logs (:error, "SBML reported error: File unreadable.") @test_throws AssertionError readSBML(
+        sbmlfile * ".does.not.really.exist",
```

Comment:
nitpick but why is the error message `.does.not.really.exist` with the periods?

Reply:
that's not the mesage.

Reply:
ok u can resolve this then

Reply:
oh yeah totally misread that